package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "stylelint-create",
+  "version": "0.0.1",
+  "description": "Initialization wizard for Stylelint, a linter that helps you avoid errors and enforce conventions in your styles.",
+  "keywords": [
+    "css-in-js",
+    "css",
+    "less",
+    "lint",
+    "linter",
+    "markdown",
+    "sass",
+    "scss",
+    "stylelint",
+    "sugarss"
+  ],
+  "homepage": "https://github.com/stylelint/stylelint-create#readme",
+  "bugs": {
+    "url": "https://github.com/stylelint/stylelint-create/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/stylelint/stylelint-create.git"
+  },
+  "funding": {
+    "type": "opencollective",
+    "url": "https://opencollective.com/stylelint"
+  },
+  "license": "MIT",
+  "author": "stylelint",
+  "main": "create-stylelint.mjs",
+  "bin": {
+    "create-stylelint": "./create-stylelint.mjs"
+  },
+  "scripts": {
+    "format": "prettier . --write --cache",
```

Comment:
question

Would `--ignore-path .gitignore` be necessary?

Reply:
Is your concern `node_modules`? If so, that's handled [by default](https://prettier.io/docs/en/ignore.html):

> By default prettier ignores files in version control systems directories (".git", ".svn" and ".hg") and node_modules (if [--with-node-modules CLI option](https://prettier.io/docs/en/cli.html#--with-node-modules) not specified)